code/index.html

```diff
@@ -1,12 +1,19 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Project 4: Pizzabot</title>
-  </head>
-  <body>
-    <h1>This is your Pizza Bot</h1>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <title>Project 4: Pizzabot</title>
+</head>
+
+<body>
+  <h1>This is your Pizza Bot</h1>
+  <h2 id="message"></h2>
+  <button id="orderButton" onclick="clicked()">Order</button>
```

Comment:
It would be good if the function name would be more describing of what is does. `clicked()` doesn't say so much of what will happen in that function